packages/app/README.md

```diff
@@ -389,6 +389,40 @@ import { getFeedbackMailto } from '@h5web/app';
 }} />
 ```
 
+#### `enableBigIntSerialization`
+
+Invoke this function before rendering your application to allow the _Raw_
+visualization and metadata inspector to serialize and display
+[big integers](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#bigint_type):
+
+```jsx
+enableBigIntSerialization();
+createRoot(document.querySelector('#root')).render(<MyApp />);
+```
+
+This is recommended if you're working with a provider that supports 64-bit
+integers — i.e. one that may provide dataset and attribute values that include
+primitive `bigint` numbers.
+
+The _Raw_ visualization and metadata inspector rely on `JSON.stringify()` to
+render dataset and attribute values. By default, `JSON.stringify()` does not
+know how to serialize `bigint` numbers and throws an error if it encounters one.
+`enableBigIntSerialization()` teaches `JSON.stringify()` to convert big integers
+to strings:
```

Comment:
I think what you wrote in the PR description is worth mentioning: 

> if H5Web is integrated into an app that already implements its own JSON serialization, H5Web could unknowingly break this parent app 

Corollary: if your app already implements `BigInt` serialization, `enableBigIntSerialization` is not needed